### PR TITLE
[OrderByComparator] LPD 28067 Apply Singleton in module commerce-price-list-api

### PR DESCRIPTION
**Pull request by Valmir-unicap**
- Ticket: https://liferay.atlassian.net/browse/LPD-28067
- Correction of: https://github.com/liferay-core-infra/liferay-portal/pull/7183

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:relevant

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -1,4 +1,76 @@
 [
+	{
+		"from": "new CommercePriceEntryUOMCreateDateComparator()",
+		"issueKey": "LPD 28067",
```
Comment (dantewang) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:4]: @Valmir-unicap The issue key here should have a `-` e.g. `LPD-28067`.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava [hunk 2]
```diff
@@ -64,6 +64,13 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		AssetListEntryCreateDateComparator.getInstance(false);
 		AssetTagCountComparator.getInstance(false);
 
+		CommercePricingClassCreateDateComparator.getInstance(false);
+        CommerceTierPriceEntryMinQuantityComparator.getInstance(false);
```
Comment (dantewang) [inline modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava:68]: @Valmir-unicap several wrong spacing in these lines, we need to use tabs not spaces.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
@Valmir-unicap Looks good, please just fix the SF issues.

**Comment by dantewang** [comment]
@Valmir-unicap By the way the compile error seems relevant:

```
     [exec] > Task :apps:commerce:commerce-frontend-taglib:compileJava FROM-CACHE
     [exec] > Task :apps:commerce:commerce-frontend-taglib:copyLibs SKIPPED
     [exec] > Task :apps:commerce:commerce-frontend-taglib:classes
     [exec] /opt/dev/projects/github/liferay-portal/modules/apps/commerce/commerce-taglib/src/main/java/com/liferay/commerce/taglib/servlet/taglib/TierPriceTag.java:65: error: cannot access com.liferay.petra.sql.dsl.query.sort.OrderByInfo
     [exec] 							getCommerceTierPriceEntries(
     [exec] 							                           ^
     [exec]   class file for com.liferay.petra.sql.dsl.query.sort.OrderByInfo not found
     [exec] 1 error
     [exec] 
     [exec] > Task :apps:commerce:commerce-taglib:compileJava FAILED
```

Please resend.

**Comment by Valmir-unicap** [comment]
Thanks @dantewang!  I'm going to fix it!

**Pull request closed without merging**